README.rst

```diff
@@ -75,12 +75,29 @@ You can also use ``tox`` to setup a clean and disposable environment::
 
     $ tox -e py3.4  # Runs tests with Python 3.4
 
-To run tests for all environments, lint and documentation tests,
+To run Python tests for all environments, lint and documentation tests,
 use simply ``tox``. This simulates what our CI also does.
 
 To run linting tests (pep8 and static code analysis), use ``tox -e lint`` or
 ``make lint``.
 
+Kolibri comes with a Javascript test suite based on ``mocha``. To run all tests::
+
+    $ npm test
```

Comment:
As npm is only used for dev installs, I have made no distinction between dev and non-dev dependencies.